Feature/support mga v4

goncer opened the pull request.

devdanco commented:
Everything looks good but when you will merge this branch remove those console logs. Thanks

goncer commented:
@devdanco , fixed all the tests! versions differ ( left them with the weird strings until we have the proper version release ). 
+ Fixed the create token method. Think that does no longer returns a subscription. 
Plz check test/utility.ts
